composer.json

```diff
@@ -10,21 +12,22 @@
         "phpstan/phpstan-shim": "^0.9.2",
         "phpunit/phpunit": "^7.0",
         "symfony/process": "^4.0",
-        "keboola/datadir-tests": "^1.0"
+        "keboola/datadir-tests": "^2.0"
```

Comment:
@ondrejhlavacek Jak se to stalo? V template je už verze 2! Máme tam nějaký bug v tom postupu?  https://github.com/keboola/component-generator/blob/master/templates/php-component/composer.json#L13

Reply:
není to releasnuté ten generátor.

Reply:
Aha! hernajs!

Reply:
https://github.com/keboola/component-generator/releases

Reply:
Tak já releasnu novou verzi.

Reply:
Nebo možná ideálně @ondrejhlavacek by to mohl vyzkoušet tady na tom a když to projde v pohodě, tak releasnou, co?